packages/react/src/themes/teams-high-contrast/components/Menu/menuVariables.ts

```diff
@@ -1,13 +1,26 @@
+import { pxToRem } from '../../../../lib'
+
 export interface MenuVariables {
   color: string
   activeColor: string
   activeBackgroundColor: string
+  selectedBackgroundColor: string
+
+  verticalMenuBackgroundColor: string
+  menuDividerHeight: string
```

Comment:
NIT: ~~`menuDividerHeight`~~ -> `dividerHeight` since we're already in the context of the menu